bundles/org.eclipse.equinox.cm.test/META-INF/MANIFEST.MF

```diff
@@ -17,4 +17,3 @@ Import-Package: org.eclipse.core.runtime,
 Bundle-ActivationPolicy: lazy
 Bundle-RequiredExecutionEnvironment: JavaSE-17
 Automatic-Module-Name: org.eclipse.equinox.cm.test
-Require-Bundle: org.eclipse.equinox.cm
```

Comment:
Is this maybe only required to pull in the implementation for the test? Maybe a require capability would be better!